config.rb

```diff
@@ -49,10 +49,26 @@
 # Build-specific configuration
 configure :build do
   # Minify CSS on build
-  # activate :minify_css
+  activate :minify_css
 
   # Minify Javascript on build
-  # activate :minify_javascript
+  activate :minify_javascript
+
+  # Hash assets on build
+  activate :asset_hash
+end
+
+# The following lines copies files from one place to another
+# Because govuk-frontent fonts and images are with node_modules so they aren’t _seen_ by sprockets
+fonts_dir = Dir[File.join(root, "node_modules/govuk-frontend/assets/fonts/*")]
+fonts_dir.each do |filename|
+  dest_folder = File.join(root, "source/assets/fonts")
```

Comment:
This falls over if the font doesn't already exist (e.g. on a clean install)

Reply:
Ah good point, will do a `.gitkeep` on the folders